apps/ideaspace/src/components/modules/SubmissionForm/SubmissionForm.js

```diff
@@ -32,13 +32,13 @@ function SubmissionForm() {
   );
 
   const getDBInvolveLevel = (involveString) => {
-    if (involveString.includes('and also believe')) {
+    if (involveString.includes('to end')) {
```

Comment:
This looks hacky. Can we change the backend to store this as an enum?

Reply:
@chungthuang  We can, but seems out of scope for this frontend change. We would still need to map the values to an enum.

Do you have a solution in mind that avoids making a comparison like this to get a value to save to the database?

Reply:
@chungthuang just saw your other comment and updated this